tests/test_http_response.py

```diff
@@ -343,6 +343,21 @@ def test_unicode_body(self):
         self.assertTrue(isinstance(r1.text, str))
         self.assertEqual(r1.text, unicode_string)
 
+    def test_json(self):
+        body = u"[{\"id\": \"1234\", \"type\": \"event\"}]"
```

Comment:
```suggestion
        body = """[{"id": "1234", "type": "event"}]"""
```
A little bit cleaner this way